include/vkgcDefs.h

```diff
@@ -1307,11 +1307,29 @@ static constexpr unsigned RayTracingMaxShaderNameLength = 16;
 /// Raytracing invalid shader ID
 static constexpr uint64_t RayTracingInvalidShaderId = 0;
 
+/// The mapping method that the driver should apply to the shader IDs returned by the compiler in
+/// RayTracingShaderIdentifier structures.
+enum class RayTracingShaderIdentifierMapping : unsigned char {
+  /// Interpret the ID as an index into the ELF module array and produce the full GPU VA of its unique contained
+  /// function.
+  ElfModuleGpuVa = 0,
+
+  /// Interpret the ID as an index into the ELF module array and produce the low 32 bits of the GPU VA of its unique
+  /// contained function. The high 32 bits remain 0 (but extraBits are still applied if present).
+  ElfModuleGpuVaLo,
+
+  /// Interpret the ID as an index into the ELF module array and produce the low 32 bits of the GPU VA of its unique
+  /// contained function. The high 32 bits are applied with extraBits
+  ElfModuleGpuVaHi,
```

Comment:
As discussed internally, I don't think this option makes sense.

Reply:
To be clear, I think we should just make shaderIdExtraBits an `uint64_t` and always OR it into the VA, for both ElfModuleGpuVa and ElfModuleGpuVaLo. And then perhaps shaderIdMapping can be changed to just a `bool onlyGpuVaLo`.

Reply:
If the gpu va is Or with shaderIdExtraBits in all case,  then perhaps the shaderIdMapping  is not used/required.

Reply:
There's still the future possibility that we want to see the high bits of the VA but stuff some metadata into LSBs, or maybe into the top 16 bits (assuming 48 bits of virtual address space).